doc/tekniker_info.rst

```diff
@@ -0,0 +1,108 @@
+.. py:currentmodule:: lsst.ts.MTMount
+
+.. _lsst.ts.MTMount-Tekniker_info:
+
+Interactions with Tekniker's software
+=====================================
+
+Acronyms
```

Comment:
I have something similiar listed here. https://tstn-008.lsst.io/acronyms.html

I don't mind keeping Tekniker info listed here. I only wish to point out that the technote 008 is for TMA related documentation and perhaps we can list the notes there and reference to them from here. Just an idea, I will approve regardless and let it be your decision.

Reply:
It is an interesting idea. I compared the lists and thought about it, and decided to keep the lists separate because they exist for very different purposes:
* The list in ts_MTMount describes the acronyms used by Tekniker in their source code. It contains all acronyms used the COMMAND_TYPE enums in types.h. It must track Tekniker's code.
* The list in TN-008 describes abbreviations used the tech note. There is a lot of overlap, but some differences, which would be a nuisance.

If you think it would be useful to have my table in TN 008 then we can consider adding it. But I fear it would be confusing due to the mismatch with abbreviations used in the document.